packages/babel-helper-explode-assignable-expression/src/index.ts

```diff
@@ -28,7 +36,7 @@ function getObjRef(node, nodes, file, scope) {
       return ref;
     }
   } else {
-    throw new Error(`We can't explode this node type ${node.type}`);
+    throw new Error(`We can't explode this node type ${node["type"]}`);
```

Comment:
Why?

Reply:
Otherwise TS will think this branch is unreachable and `type` property is not defined in `never`.